cli/components/list.go

```diff
@@ -39,8 +39,9 @@ func (d itemDelegate) Render(w io.Writer, m list.Model, index int, listItem list
 
 	fn := itemStyle.Render
 	if index == m.Index() {
-		fn = func(s string) string {
-			return selectedItemStyle.Render("> " + s)
+		fn = func(strs ...string) string {
+			strs = append([]string{"> "}, strs...)
```

Comment:
The renderer changed. It now looks like this:
![image](https://github.com/mondoohq/cnquery/assets/827818/c09d6f93-b0ee-4a05-be1b-0ea73bc2044a)

The selected item is now intended one space. When we want to keep it as it was before, we would need to remove the space in this line.

Reply:
We should remove that and keep the current behavior.

Reply:
Thank you @czunker I updated and fixed this issue